### ISSUE TITLE
[Datepicker]The "aria-label" for  "next month" button does not work

### ISSUE THREAD

**Issue by AllForNothing** [vmware-clarity/ng-clarity#450]
**This is a bug report for the `@clr` Angular or UI versions of the design system.
For the web-component implementation of Clarity (`@cds`), visit [vmware-clarity/core](https://github.com/vmware-clarity/core).**

## Describe the bug
The "Next month"  text will not appear when the mouse hovers on the "next month" button:
<img width="1455" alt="image" src="https://user-images.githubusercontent.com/30999793/211247836-a9f290bd-a2af-472b-8286-73fd0a07b360.png">
This is really odd.

But it works when I change "aria-label" to "title":
<img width="1445" alt="image" src="https://user-images.githubusercontent.com/30999793/211248197-f7ca6c59-b11b-4666-a8d5-495eb6c9d438.png">

## How to reproduce

_Please provide a link to a reproduction scenario using one of the [Clarity Stackblitz templates](https://stackblitz.com/@clr-team). Reports without a clear reproduction may not be prioritized until one is provided._

Run the code of the main branch, then hover on the "next month" button

## Expected behavior

"Next month" should appear

## Versions

**Clarity version:**

latest

**Framework version:**
_ie: Angular 15_

**Device:**

Tested on  chrome, firefox, and safari of macbook

## Additional notes

_Add any other notes about the problem here._

**Comment by kevinbuhmann** [comment]
The `aria-label` is needed for screen readers. Some screen readers do not read `title` attribute, so we cannot replace `aria-label` with `title`.

And my understanding is that the browser `title` is not an accessible tooltip, so we generally avoid it.

We could use a more accessible tooltip (like `clr-tooltip), but that seems unnecessary here. The next month button in the datepicker follows a very common/intuitive pattern, so users know what it does without explanation.

I will cc our comtact on VMware's accessibility team for confirmation and/or more information.

cc: @AmyLiNow

**Comment by kevinbuhmann** [comment]
The `aria-label` is needed for screen readers. Some screen readers do not read `title` attribute, so we cannot replace `aria-label` with `title`.

And my understanding is that the browser `title` is not an accessible tooltip, so we generally avoid it.

We could use `clr-tooltip` which is more accessible than the `title` attribute, but that seems unnecessary here. The next month button in the datepicker follows a very common/intuitive pattern, so users know what it does without explanation.

I will cc our contact on VMware's accessibility team for confirmation and/or more information.

cc: @AmyLiNow

**Comment by williamernest** [comment]
I don't think this is advocating for one or the other, but more of an observation that the buttons aren't consistent. The previous month and current month buttons use title on the icon as well as aria label on the button. The next month button adds aria-label to the button but does not add the title on the icon. The title for the icon shows as a tooltip on some (maybe all?) browsers when you hover over it, which is leading to this inconsistency where if you leave your cursor over last/current month you get a small browser tooltip, but not for the next month. 

I think the approach here is to either add the title attribute to all 3 cds-icon elements or remove them all so we're consistent. 

https://github.com/vmware-clarity/ng-clarity/blob/54973e889a68770414690650553f5518d55b3be8/projects/angular/src/forms/datepicker/daypicker.html#L46

![image](https://user-images.githubusercontent.com/34519388/222606235-d064858e-7b97-4913-ab7d-de18c6e0ea7e.png)
